includes/class-wsuwp-graduate-degree-programs.php

```diff
@@ -270,52 +307,256 @@ public function display_factsheet_primary_meta_box( $post ) {
 				$data[ $key ] = array( false );
 			}
 
-			if ( isset( $meta['pre_html'] ) ) {
-				echo $meta['pre_html']; // @codingStandardsIgnoreLine (HTML is static in code)
+			if ( 'primary' !== $meta['location'] ) {
+				continue;
+			}
+
+			$this->output_meta_box_html( $meta, $data, $key );
+		}
+
+		echo '</div>'; // End factsheet-primary-inputs.
+	}
+
+	/**
+	 * Displays a meta box to capture faculty member information for
+	 * a factsheet.
+	 *
+	 * @since 0.7.0
+	 *
+	 * @param WP_Post $post
+	 */
+	public function display_faculty_meta_box( $post ) {
+		$faculty_members = wp_get_object_terms( $post->ID, 'gs-faculty' );
+		$faculty_relationships = get_post_meta( $post->ID, 'gsdp_faculty_relationships', true );
+
+		foreach ( $faculty_members as $faculty_member ) {
```

Comment:
I'm probably just missing something, but even with a few Faculty terms, nothing shows up in this meta box for me.